test/cookbooks/python-webapp-test/recipes/default.rb

```diff
@@ -15,3 +15,26 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+package 'git'
+
+settings_hash = {
+  host: 'db.example.com',
+  port: '5432',
+  username: 'user',
+  password: 'pass',
+  db_name: 'database',
+  secret_key: 'abcd'
+}
+
+python_webapp_common 'whats_fresh' do
+  create_user true
+  owner 'whats_fresh'
+  group 'whats_fresh'
+
```

Comment:
This is pretty close to a maximal integration test case already. The only things this does NOT change from the defaults are ``path``, ``revision``, and ``virtualenv_path``. Perhaps consider adding those to this test, and adding another test that tests the default cases?